test-apps/presentation-test-app/package.json

```diff
@@ -55,7 +55,8 @@
     "react-dom": "^17.0.0",
     "react-select": "3.2.0",
     "react-resize-detector": "^6.7.6",
-    "semver": "^5.5.0"
+    "semver": "^5.5.0",
+    "mocha": "~10.0.0"
```

Comment:
Why do test-apps need to have a dependency on `mocha`? I know that there are no tests at least in `presentation-test-app`.